### ISSUE TITLE
Researching multiple tech nodes progress simultaneously

### ISSUE THREAD

**Issue by hoylemd** [magico13/KCT#121]
When I choose more than one tech node to unlock, they both seem to be researched at the same rate.  Would it be possible to have a research queue and multiple research 'teams' like we have now with building?

**Comment by magico13** [comment]
You can, to a degree, by modifying the [Node Formula](https://github.com/magico13/KCT/wiki/Preset-Formulas-and-Math-Parser-Documentation). You still only have the one rate that you can put points into, but the second, third, etc. rates can be some multiple of that first rate. I believe RP-0 uses this to have just a single rate.

Example:
"2^([N]+1) / 86400 \* sign(-[I])" would have a single rate.
"2^([N]+1) / 86400 \* sign(1-[I]) / ([I]+1)" would have two rates where the second rate is half as fast as the first.

As soon as you stop having all rates the same KCT will provide some buttons to let you reorder the list.

**Comment by hoylemd** [comment]
Ah, so I'd have to go into the presets to do this, and I'd only have control over the base rate in-game? That's closer to what I had in mind, but maybe in the future it would be good to have research 'teams' the same way you have build 'teams' (Thats how I think of the build rates - different teams working on different projects).

**Comment by magico13** [comment]
Agreed. I'll try to think of a good way of implementing that in a modular way since I'd like to keep the default Preset as "all at once". I've got some other things I want to change regarding how upgrades are handled, so I can try to add that in at the same time.

That won't be for a while though, so until then you're stuck with the current method.

**Comment by hoylemd** [comment]
Cool. How do you track feature requests?  Just leave this issue open, or do you track it elsewhere?

**Comment by magico13** [comment]
Just leave the issue open. I use GitHub issues as basically a todo list to remind me of what I need to work on when I've got time. I'll close it once I've got it implemented and merged into Master (aka, a full release).

**Comment by hoylemd** [comment]
great.  Maybe I'll take a stab at it sometime and submit a PR :)
